firebase-sessions/src/main/kotlin/com/google/firebase/sessions/FirebaseSessions.kt

```diff
@@ -33,14 +33,18 @@ internal constructor(
 ) {
   private val sessionGenerator = SessionGenerator(collectEvents = true)
   private val sessionCoordinator = SessionCoordinator(firebaseInstallations, backgroundDispatcher)
+  private val firebaseApp = firebaseApp
```

Comment:
Kotlin has a shortcut for this, you can just add `val` to the firebaseApp in the primary constructor.

Reply:
Nice comfort with Kotlin. Done.